### ISSUE TITLE
Rename repository to `dart-lang/website` or `dart-lang/dart.dev`

### ISSUE THREAD

**Issue by parlough** [dart-lang/site-www#5221]
### Describe the problem

`site-www` is a rather unguessable name, especially for newer developers who may have experienced `www` being mostly missing from their browser experience for the past decade. We should consider a more universally standard or understandable repository name. This will allow for easier discovery of the repository and consistency with elsewhere.

Possibilities (and discussion):
- 🟢 `dart-lang/website` - Short, simple, and consistent. Doesn't need to change with new domain. Examples: [docs.flutter.dev](https://github.com/flutter/website), [go.dev](https://github.com/golang/website), [kubernetes.io](https://github.com/kubernetes/website), [vitess.io](https://github.com/vitessio/website), [etcd.io](https://github.com/etcd-io/website), [electronjs.org](https://github.com/electron/website)
- 🟡  `dart-lang/dart.dev` - This seems popular and is very searchable but period is potentially confusing in links and for some tooling. Also might end up with issues from subdomains (like `api.dart.dev`) Examples: [react.dev](https://github.com/reactjs/react.dev), [istio.io](https://github.com/istio/istio.io), [lit.dev](https://github.com/lit/lit.dev), [haxe.org](https://github.com/HaxeFoundation/haxe.org), [astro.build](https://github.com/withastro/astro.build)
- 🟠 `dart-lang/docs` - Users could expect this repository to contain the API docs. Examples: [tensorflow.org](https://github.com/tensorflow/docs), [vuejs.org](https://github.com/vuejs/docs), [.NET](https://github.com/dotnet/docs), [docs.astro.build](https://github.com/withastro/docs)
- 🟠 `dart-lang/dart-site` - It's so so, but not common and a bit repetitive since `dart` is already in the organization.
- 🔴 `dart-lang/dart-dev` - Similar to current `dart-lang/pub-dev`, but it's not clear what this repository is about. Could be seen as the `dart` tool from the name.
- ❓ Others - https://github.com/microsoft/TypeScript-Website, https://github.com/microsoft/vscode-docs

### Expected fix

- [ ] Rename repository to `dart-lang/website` (or whatever is chosen). GitHub redirects links to the old destination
- [ ] Update any internal repo tracking/tooling for repository rename
- [ ] Update primary text and links across various Dart and Flutter repos/docs
- [ ] Slowly update other old links as came across (redirect will cover those cases)

**Comment by parlough** [comment]
> Hello @parlough, can I work on this issue?

Hey @shaikahmadnawaz! Thanks for the offer. 

However, this particular proposal still needs to be reviewed/approved and would need to be at least initially worked on by an admin in the `dart-lang` GitHub organization.

If you have feedback on a better name, that would of course be welcome :)

**Comment by shaikahmadnawaz** [comment]
> > Hello @parlough, can I work on this issue?
> 
> Hey @shaikahmadnawaz! Thanks for the offer. 
> 
> However, this particular proposal still needs to be reviewed/approved and would need to be at least initially worked on by an admin in the `dart-lang` GitHub organization.
> 
> If you have feedback on a better name, that would of course be welcome :)   

Okay 👍🏻

**Comment by atsansone** [comment]
The only issue I see with `dart-lang/website` is that it can make working on multiple repos difficult if both display "website". I'm okay with `site-www` for now, though we could revisit it and maybe `flutter` as well.

I think `dart-lang/dart.dev` could be a winner.

@mit-mit : Any thoughts?

**Comment by shaikahmadnawaz** [comment]
> The only issue I see with `dart-lang/website` is that it can make working on multiple repos difficult if both display "website". I'm okay with `site-www` for now, though we could revisit it and maybe `flutter` as well.
> 
> I think `dart-lang/dart.dev` could be a winner.
> 
> @mit-mit : Any thoughts?

Agreed, dart.dev sounds like a suitable choice @atsansone @parlough

**Comment by parlough** [comment]
Thanks for your thoughts! 

I do slightly prefer `dart-lang/website` (for the reasons noted earlier) but I'm also cool with `dart-lang/dart.dev`.

As a note for everyone, this issue is extremely low priority, but I opened it to collect feedback.

**Comment by mit-mit** [comment]
I also slightly prefer `/website`. We generally have several cases where repo names are aligned across the Dart and Flutter github orgs.

**Comment by droidbg** [comment]
Adding my thoughts:
I prefer `dart-lang/dart.dev` over `dart-lang/website` , as it is more clear which website this repository covers. 

- `dart-lang/dart.dev`: This name suggests that the repository is specifically for the Dart programming language's official website.
- `dart-lang/website` : This name is more generic and suggests that the repository contains the code for a website related to the Dart programming language but not necessarily the official website. Also It can happen that people can confuse just `dart-lang/website` for `pub.dev website`. 

Or a combination for both is also good, something like: 
- `dart-lang/dart-website`: This combines both elements, specifying that it's related to Dart and a website. It provides a clear description of the repository's purpose.

**Comment by parlough** [comment]
I'd really like to do this. I do think the name makes the repository harder to discover and it's just something else for people to remember.

Are there any known technical reasons this can't be updated to `dart-lang/website`? If not, I propose we just make the transition. GitHub maintains redirects and I'll happily handle updating references here and elsewhere so they are more accurate.

**Comment by mit-mit** [comment]
`dart-lang/website` SGTM

@atsansone any objections?

**Comment by parlough** [comment]
I discussed with Tony and he's okay with the change as long as I followup with some contribution doc adjustments.

No rush on these, but the update process will be something like:

1. Update Cloud Build triggers for new nam
2. Rename repository
3. Land updates for new repository name (https://github.com/dart-lang/site-www/pull/5492)

I'll start by touching base with Khanh on any necessary Cloud Build changes.

**Comment by antfitch** [comment]
This sounds like an awesome change. I hope we see it this year!

**Comment by mit-mit** [comment]
Yeah, let's do it. The current name made sense back when we had multiple sites, but not so much anymore.
